backend/apps/owasp/admin/entity_member.py

```
@@ -66,15 +75,24 @@ def entity(self, obj):
     def get_search_results(self, request, queryset, search_term):
-        """Get search results from entity name or key."""
+        """
+        Extend default search to also match entity names and keys.
+
+        In addition to the built-in search, this method searches:
+        - Project name or key
+        - Chapter name or key
+        - Committee name or key
+
+        and includes matching EntityMember rows in the results.
+        """
```

Ruff (0.14.5)

Multi-line docstring summary should start at the first line

Remove whitespace after opening quotes

(D212)